### PR TITLE
DL-6440, Session updates moved into GuidanceService and Init fns moved into RetrieveAndCache service

### PR DESCRIPTION
**Pull request by cmrhmrc**

### CODE REVIEW

File: app/services/SessionService.scala [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+// import repositories.SessionRepository
+// import core.models.ocelot.{LabelCache, Labels, Process}
+
+// @Singleton
+// class SessionService @Inject() (
+//     appConfig: AppConfig,
+//     sessionRepository: SessionRepository,
+//     messagesApi: MessagesApi
+// ) {
+
```
Comment (paulanderson1234) [inline app/services/SessionService.scala]: Any value in this file if everything is commented out?

Comment (cmrhmrc) [reply app/services/SessionService.scala]: This file is deleted

File: test/models/ProcessContextSpec.scala [hunk 2]
```diff
@@ -23,7 +23,7 @@ import play.api.inject.Injector
 import org.scalatestplus.play.guice.GuiceOneAppPerSuite
 import services.SecuredProcessBuilder
 
-class ProcessContextSpec extends BaseSpec with GuiceOneAppPerSuite {
+class GuidanceSessionSpec extends BaseSpec with GuiceOneAppPerSuite {
```
Comment (paulanderson1234) [inline test/models/ProcessContextSpec.scala:26]: Would we not rename this file instead of leaving it as ProcessContextSpec' ?

Comment (cmrhmrc) [reply test/models/ProcessContextSpec.scala:26]: renamed and pushed to branch

File: app/services/SessionService.scala (hunk 1 above)
Comment (paulanderson1234) [reply app/services/SessionService.scala]: Confirmed - resolved

File: test/models/ProcessContextSpec.scala (hunk 2 above)
Comment (paulanderson1234) [reply test/models/ProcessContextSpec.scala:26]: Resolved

**Pull request merged**